collector/systemd_linux.go

```diff
@@ -424,3 +445,23 @@ func filterUnits(units []unit, whitelistPattern, blacklistPattern *regexp.Regexp
 
 	return filtered
 }
+
+func getSystemdVersion() int {
+	conn, err := newSystemdDbusConn()
+	if err != nil {
+		log.Warnf("Unable to get systemd version, defaulting to 0: %s", err)
```

Comment:
This should probably be "Unable to get systemd dbus connection"

Reply:
Done